pkg/vault/reconcile.go

```diff
@@ -27,7 +30,11 @@ func DiffItems(desired, existing []Item) (toBeWritten, toBeDeleted []Item) {
 				toBeWritten = append(toBeWritten, item)
 			}
 		}
-
+		for _, item := range desired {
+			if !descriptionIn(item, existing) {
```

Comment:
I think there needs to be another check here for options as well. If description OR options have changed, then an tuning update needs to be applied.